tools/nvqpp/nvq++.in

```diff
@@ -375,6 +381,11 @@ while [ $# -ne 0 ]; do
 	--llvm-version=*)
 		llvm_suffix="${arg#*=}"
 		;;
+	--gcc-toolchain=*|--gcc-install-dir=*)
+		# clang is deprecating the --gcc-toolchain option
+		CUDAQ_QUAKE_ARGS="${CUDAQ_QUAKE_ARGS} -Xcudaq \"$1\""
```

Comment:
Given the change in https://github.com/NVIDIA/cuda-quantum/pull/1015/commits/f0c56560786a11c826ad182174520aae44989f8f, it might be worth removing these `\"`'s too.